clang/lib/AST/DeclTemplate.cpp

```diff
@@ -244,6 +245,17 @@ bool TemplateParameterList::hasAssociatedConstraints() const {
   return HasRequiresClause || HasConstrainedParameters;
 }
 
+ArrayRef<TemplateArgument> TemplateParameterList::getInjectedTemplateArgs() {
+  if (const auto *Context = InjectedArgs.dyn_cast<const ASTContext *>()) {
+    TemplateArgument *Args = new (*Context) TemplateArgument[size()];
```

Comment:
Do we ensure this gets deleted somewhere?  It seems to me we need to make sure that this gets cleaned up.